web-common/src/features/scheduled-reports/time-utils.ts

```diff
@@ -119,7 +119,9 @@ export function getTimeOfDayFromCronExpression(cronExpr: string): string {
 
 export function getDayOfMonthFromCronExpression(cronExpr: string): number {
   const [, , dayOfMonth] = cronExpr.split(" ");
-  return Number(dayOfMonth);
+  const dayOfMonthAsNum = Number(dayOfMonth);
+  if (Number.isNaN(dayOfMonthAsNum)) return 1;
```

Comment:
If `dayOfMonth === "*"`, which would resolve to `NaN`, then it feels wrong to consider it the same as `1`?

Reply:
It doesnt really get used if the report is not monthly. Also since the Day-Of-Month selector is disabled there is no way to change it either. So lets change it when we do support Day-Of-Month selection.

Reply:
Ok, yeah, mind just adding a code comment to explain this?